drools-game-engine-services/src/main/java/org/drools/game/services/endpoint/impl/CommandRest.java

```diff
@@ -0,0 +1,29 @@
+package org.drools.game.services.endpoint.impl;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.CLASS;
+
+import org.drools.game.core.api.Command;
+import org.drools.game.model.api.Player;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = CLASS, include = WRAPPER_OBJECT)
+public class CommandRest<T> {
```

Comment:
Do we really need this? Why not send a Command directly instead of having a wrapper for the command?